pkg/network/setup/network.go

```diff
@@ -58,12 +65,17 @@ func (v VMNetworkConfigurator) getPhase2NICs(domain *api.Domain, networks []v1.N
 			return nil, fmt.Errorf("no iface matching with network %s", networks[i].Name)
 		}
 
-		// Binding plugin, SR-IOV and Slirp devices are not part of the phases
-		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
+		// Binding plugin (with non tap domain attachment), SR-IOV and Slirp devices are not part of the phases
+		bindingType, _ := v.domainAttachments[iface.Name]
+		if (iface.Binding != nil && bindingType != string(v1.Tap)) || iface.SRIOV != nil || iface.Slirp != nil {
 			continue
```

Comment:
```suggestion
bindingType := v.domainAttachments[iface.Name]
nonTapBinding := iface.Binding != nil && bindingType != string(v1.Tap)
if nonTapBinding || iface.SRIOV != nil || iface.Slirp != nil {
```

If `domainAttachments` is nil or interface name no exist, it will return the default value, in this case empty string.